src/components/Alert.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from 'react'
-import { Alert as BsAlert } from 'react-bootstrap'
+import BsAlert from 'react-bootstrap/Alert'
```

Comment:
This does not seem quite right. `Alert` is imported as `BsAlert` as our own component is named `Alert`.

Reply:
Should I name it differently? Maybe `RBAlert`?

Reply:
No, not at all. I'm fine with that. It's also fine if we don't merge this at all :)

Reply:
I see you point @theborakompanioni, `rb.*` definitely increases readability. But on the other hand if this is the recommended approach from the library authors, I'd trust that and go with the explicit component imports. We just need to be careful not to confuse our own components with the ones from Bootstrap. No super strong feelings towards either approach though, just a feeling.

Reply:
Having thought it through again, I am in favor of not merging. And after the thorough analysis of @Djo1e, I see even less reasons to do so. Also, no super strong feelings. Just to let you know.

Reply:
Ok then let's not merge. Ok with you @Djo1e?

Reply:
Yes, closing this PR